### ISSUE TITLE
use underscores in slugs

### ISSUE THREAD

**Issue by denis-gorin** [bolt/core#2616]
Hi,
Is there any way to use underscores in slugs?

**Comment by bobdenotter** [comment]
No, symfony automatically generates them using dashes. What is the usecase for this?

**Comment by denis-gorin** [comment]
ok, I get it
usecase - just client is using predefined adresses in QR codes on their product packs

**Comment by denis-gorin** [comment]
I thought I could redefine it in config/packages/cocur_slugify.yaml
```
cocur_slugify:
    separator: '_'
```
but failed

**Comment by abdellahramadan** [comment]
Here Symfony allows the definition of separator [SluggerInterface](https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/String/Slugger/SluggerInterface.php)
The solution might be to allow a user define it as well in yaml. 
Can I try this as well @bobdenotter ? If it's necessary.
